dynawo/sources/Modeler/Common/DYNConnector.h

```diff
@@ -110,6 +103,31 @@ class connectedSubModel {
     return negated_;
   }
 
+  /**
+   * @brief Equality operator
+   *
+   * Required in order to use set container
+   * @param other
+   * @returns operator status
```

Comment:
For me the comment aims to explain why the function exists in the first place, since it is not explicitly used. This is why I say  it is used by "set" container

Reply:
yes, but the objective of the doxygen is for the developper to know what the methods does, not why it was created

Reply:
OK